File: android/core/ui/src/main/java/com/multiplayer/core/ui/tokens/MultiplayerRadius.kt
package com.multiplayer.core.ui.tokens

import androidx.compose.runtime.Immutable
import androidx.compose.ui.unit.Dp
import androidx.compose.ui.unit.dp

@Immutable
data class MultiplayerRadius(
    val small: Dp = 8.dp,
    val medium: Dp = 12.dp,
    val large: Dp = 20.dp,
    val xLarge: Dp = 28.dp,
    val pill: Dp = 999.dp,
)

File: ios/Core/UI/Sources/CoreUI/Tokens/MultiplayerRadius.swift
import CoreGraphics

public struct MultiplayerRadius {
    public let small: CGFloat
    public let medium: CGFloat
    public let large: CGFloat
    public let xLarge: CGFloat
    public let pill: CGFloat

    public init(
        small: CGFloat = 8,
        medium: CGFloat = 12,
        large: CGFloat = 20,
        xLarge: CGFloat = 28,
        pill: CGFloat = 999
    ) {
        self.small = small
        self.medium = medium
        self.large = large
        self.xLarge = xLarge
        self.pill = pill
    }
}
